stacks/cert-manager/README.md

```diff
@@ -1,26 +1,18 @@
+[Cert-Manager](https://cert-manager.io/) is an open-source certificate management tool designed to work with Kubernetes. It supports all the required operations for obtaining, renewing, and using SSL/TLS certificates. Cert-Manager can talk with various certificate authorities (CAs), like [Let's Encrypt](https://letsencrypt.org/), [HashiCorp Vault](https://www.vaultproject.io/), and [Venafi](https://www.venafi.com/). It can also automatically issue valid certificates for you and renew them before they expire.
 
-Why do you need `SSL/TLS` certificates ?
+SSL/TLS certificates secure your connections and data by verifying the identity of hosts/sites and encrypting your data. Cert-Manager manages them by integrating with your Kubernetes cluster's Ingress Controller, which is your cluster's main entry point and sits in front of its backend services. Then, you can provide identity information to users by presenting them a valid SSL/TLS certificate whenever they visit your website(s).
 
-First, and the most important aspect is verifying the `identity` of a host or site. You need to make sure that the person or company holding a website is to be trusted. This is a very important aspect, because you do not want to give your personal data (or even credit card details) to anyone.
-
-Second important aspect deals with `sensitive data` which must be `encrypted`. The communication channel should never use a simple `HTTP` scheme, which is basically a `plain-text` protocol under the hood. In other words, data must travel encrypted between the two parties (meaning you, and the website).
-
-Where does all this fit?
-
-`Cert-Manager` goes hand in hand with your `Ingress Controller` resource. An `Ingress Controller` is the main entrypoint of your `Kubernetes` cluster, and sits in front of your `backend services`. So, by securing your ingress resources (e.g. `Nginx`), sensitive data never goes in or out un-encrypted. On top of that, you provide identity information to users, by presenting them a valid SSL/TLS certificate whenever they visit your website(s). Self-signed certificates are not to be trusted, and should never be used in production systems.
-
-Please make sure to visit the Cert-Manager [official documentation](https://cert-manager.io/docs/) page to study more.
+See Cert-Manager's [official documentation](https://cert-manager.io/docs/).
 
 **Note:**
 
-DigitalOcean is using `Helm v3` to deploy `Cert-Manager` to your `DOKS` cluster.
+DigitalOcean is using Helm v3 to deploy Cert-Manager to your DOKS cluster.
```

Comment:
question: In every other PR these are hyper linked elsewhere the first time they show up, and they are also marked as `code` every where here in the PR. 

Are these code tags meant to be removed? Just want to make sure this was intended :)

Reply:
Yes, it was intended! Since these terms aren't code themselves, they shouldn't be in code spans.

Reply:
@caquillo07 This is consistent with the other edits: https://github.com/digitalocean/marketplace-kubernetes/pull/344/files#diff-09d0bac7f02052be60b7f3b55ecc4ec9866ecb6084c5e21ee41df3c51cb55d55R27 and https://github.com/digitalocean/marketplace-kubernetes/pull/330/files#diff-8a86b5d2ad6f5bd03c401797fb91fe180a96d560d8550deeaa2989fc9531644aR15.

Reply:
sounds good!